### PR TITLE
Rs 2022 08 20

### PR DESCRIPTION
**Pull request by rschlaikjer**
arach

### CODE REVIEW

File: script/Ciel/Day 6/06_01_CIEL06_4.txt [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 -- Page 161, Offset 13575.
 -- 点滴が終わって阿良句博士が屋敷を去ると、時刻は正午を回っていた。
-As the IV ends and Doctor Arach leaves the mansion, it's just past noon.
+By the time I'm done with the IV and Professor Arach has left the mansion, it's just past noon.
 }
 [sha:dec4fd7190c4b8d070a5151edfdd1dedecf1cf05]{
 -- Page 162, Offset 13576.
 -- 阿良句博士のおかげで体の調子は良くなっている。
-Thanks to Doctor Arach, my body has already started to feel better.
+Thanks to her ministrations, my body has already started feeling better.
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_4.txt:19]: mistratitions for おかげで is a _littttle_ much.
Thanks to her [nil|help|aid|care], my body has already started feeling better.

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 2]
```diff
@@ -56,22 +63,27 @@ Kohaku is... preparing breakfast. I will call Arach-sensei then. Hisui, please c
...
 -- 「阿良句先生は昨夜うちに泊まっていたんです。まだ帰っていないようだし、兄さんの体を診てもらいましょう。
-"Arach-sensei spent the night here. Since she hasn't left yet, I'll have her examine you, Nii-san.
+"Arach-sensei spent the night here.
+ She hasn't left yet, so I may as well have her examine you, Nii-san.
 }
 [sha:8cb74bf230ede658ee4ca14c88874947445531c4]{
 -- Page 125, Offset 13480.
 -- あの方、ああ見えて外科の医師でもあるんです。ここで暮らすのなら頻繁に会う人ですから、これを機に親睦を深めてください」
-Despite her appearance, she really is a doctor specializing in surgery. Now that you are living with us, you will see her frequently, so take this as an opportunity to get to know her better."
+She may not look it, but she's a doctor who specializes in surgery.
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:85]: "She may not look like it," without "look like" it sounds very un-akiha.

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 3]
```diff
@@ -126,28 +140,30 @@ Nii-san? Is something the matter? You seem restless. Could it be that, even at y
...
 -- 「はあ。では、阿良句先生が苦手とか？」
-"Haa, then is it Arach-sensei who makes you uneasy?"
+"Hmm. Then could it be that you are not comfortable around Arach-sensei?"
 }
 [sha:5711384d4b8dbbab7e11a4eb8815d58e70e21bdb]{
 -- Page 129, Offset 13493.
 -- 「苦手というかついて行けない系だろあの人は。だから、そういう話でもなくて」
 // "She's the kind of person that I find a little overwhelming"
-"I wouldn't say that she makes me uneasy, more like she's the type of person I can't really keep up with. Anyway, that's not the problem."
+"I wouldn't say I'm uncomfortable around her, it's more like she's the type of person I just can't keep up with.
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:154]: delete or update comment

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 4]
```diff
@@ -283,22 +305,35 @@ Without taking the front door, Arach-shi directly enters the living room from th
 [sha:af77e8f7bea5327d78dd2daa845f6eaf0ae52dc6]{
 -- Page 141, Offset 13522. Glued. Mods: @x, @k, @e.
 -- でも、そんなトコロが一流のサバイバーなのかしら？　情弱死すべしな世の中ですし？　目の前のご馳走をホイホイ孕ませてたら<命|タマ>がいくつあっても足りないってコトかしらー！
-But maybe that's the mark of a first-class survivor? After all, we live in a world where it's natural for those who don't know better to die, right? No matter how many <tries|b a l l s> I get, that wouldn't be enough to impregnate the snack in front of me, huh?
+// 情弱 as derog internet illiterate? Can be used for other media sources,
+// keeping it vague here since I assume it's more general in this usage.
+// Also, fucking what is going on
+ But maybe that's the mark of a first-class survivor?
+ We live in a world where you're as good as dead if you can't stay abreast of what's happening, right?
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:312]: "stay abreast of what's happening" is waaay too formal of a register for that line. "if you can't keep up with what's happening, right?"

we should just translate 情弱 as "normie" :alsoarcher:

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 5]
```diff
@@ -283,22 +305,35 @@ Without taking the front door, Arach-shi directly enters the living room from th
...
+ It doesn't matter how many lives you have,
+ if you recklessly impregnate the feast in front of you you'll never make it, will you!
 }
 [sha:d841a4ab5aacd91646592e085fa6ae4d3d355a7f]{
 -- Page 141, Offset 13523. Glued. Mods: @x.
 -- キャー、さっすが雑食動物、本能のようにヒニンしてるぅー！」
-Kyaaaa, as expected of an omnivore's instinct to avoid pregnancy!"
+// Assuming that ヒニン is intentionally kana to play on 否認
+// (deny, reject her advances, play hard to get) and 避妊 (contraception)
+ %{i}Kyaaaa%{/i}, just what I'd expect from an omnivore,
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:321]: omnivore! You've practically

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 6]
```diff
@@ -283,22 +305,35 @@ Without taking the front door, Arach-shi directly enters the living room from th
...
+ you've practically got an instinctual sense for contraception!"
 }
 [sha:26363e83a167502b52058294b3246cda4b0185e3]{
 -- Page 142, Offset 13524.
 -- などと言いながら、阿良句氏は俺の座るソファーまで歩み寄ってきた。
-And with that, Arach-sensei walks up to the sofa I am sitting on.
+As she continues to spout off like this,
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:327]: "spout off _like this_" sounds a little awkward. simplify to "go off like this" imo

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 7]
```diff
@@ -318,17 +354,25 @@ Or rather, I sure hope she isn't like this every morning.
 [sha:9aa0cea8a53b7b8ca34e4f47674a3fa72e4add3a]{
 -- Page 144, Offset 13529. Mods: @k, @e.
 -- 「はーい、それじゃあ患部を出してねー。すぐ出してねー。ちょっとサワサワするわねー。
-"Okaaay, then show me your wounds. Show them show them―. I'm going to get touchy okay―.
+"Okaaay, show me where it hurts.
+ Hurry up and whip it out for meee.
+ I'm going to get a little touchy okaaay?
```
Comment (Avuxo) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:359]: > I'm gonna get a little touchy, okaaay?

**Review by Avuxo — COMMENTED** [review]
Hmm...?
also, 256 changes on 6_3 
image

**Review by Avuxo — APPROVED** [review]

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 8]
```diff
@@ -353,92 +397,117 @@ Just by palpating my body through my clothes, she could tell how much pain I was
...
 -- この人、黙っていれば名医なんじゃないのか？
-She's actually an excellent doctor when she shuts her mouth, isn't she?
+I guess she's actually an excellent doctor?
+ So long as she keeps her mouth shut, that is.
 }
 [sha:2459b82b28e3dfe308530fa529e792662144b1a1]{
 -- Page 152, Offset 13552. Mods: @k, @e.
 -- 「んー、でも器用にケガしたものねー。肩胛骨のあたりとか、ちょっとフツーじゃないかんじ。
-"Mmm, but those wounds are pretty unconventional huh? For instance, the area around the shoulder blade doesn't look quite right.
+"Mmm, but you really seem like you got beat up by a professional, don't you?
```
Comment (yk1512) [inline script/Ciel/Day 6/06_01_CIEL06_3.txt:502]: beaten

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 6/06_01_CIEL06_3.txt [hunk 9]
```diff
@@ -182,38 +198,42 @@ It seems that I wasn't clear enough.
...
 -- 「……コホン。
-"Humph...
+"%{i}Ahem%{/i}...
 }
 [sha:5abfba9bc9fd5d7ca0627d8c96f9d0601e9ef16f]{
 -- Page 135, Offset 13507.
 -- 席を外しますから、診察が終わり次第食堂にどうぞ。
-// I'm not sure she goes to the dining room actually, change to just go to the dining room if not
-I will take my leave. Please feel free to join me in the dining room when you're done with your check-up.
+I will take my leave. Please feel free to head to the dining room once you're done with your checkup.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 6/06_01_CIEL06_3.txt:228]: Internet seems to suggest no hyphen

**Pull request merged**